docs/install/network_options.md

```diff
@@ -30,34 +30,13 @@ cp rke2-canal-config.yml /var/lib/rancher/rke2/server/manifests/
 
 For more information about the full options of the Canal config please refer to the [rke2-charts](https://github.com/rancher/rke2-charts/blob/main-source/packages/rke2-canal/charts/values.yaml).
 
-# Using Cilium instead of Canal
+# Using Cilium or Calico instead of Canal
 
-To use Cilium, Canal needs to be disabled on deployment. Refer to [Advanced Options and Configuration](../advanced.md#Disabling Server Charts) for information on how to do this.
+Starting with RKE2 v1.21, we can deploy Cilium or Calico as CNI plugin instead of Canal. To do so, pass `cilium` or `calico` as the value to the `--cni` flag. To override the default options, use a HelmChartConfig resource, as explained in the previous section for Canal options.
```

Comment:
```suggestion
Starting with RKE2 v1.21, Cilium or Calico CNI Plugins can be deployed instead of Canal. To do so, pass `cilium` or `calico` as the value of the `--cni` flag. To override the default options, use a HelmChartConfig resource, as explained in the previous section. Note that the HelmChartConfig resource names must match the chart names for your selected CNI - `rke2-cilium`, `rke2-calico`, etc.
```